src/main/java/io/github/fabriccommunity/everything/dd84ace17f2a41b6ad472b125442f384.java

```diff
@@ -0,0 +1,46 @@
+package io.github.fabriccommunity.everything;
+
+import com.mojang.blaze3d.platform.GlStateManager;
+import io.github.fabriccommunity.everything.event.c3cc5cc10df7471ebb0343d01d6d0afe;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.explosion.Explosion;
+
+import java.util.HashMap;
+
+public class dd84ace17f2a41b6ad472b125442f384 {
+	private static void e0488fa68c5b40ce852afd787da2b322() {
+		// VGhpcyBpcyBhbiBleGFtcGxlIGltcGxlbWVudGF0aW9uIG9mIGFuIGV2ZW50Lg==
+		c3cc5cc10df7471ebb0343d01d6d0afe.dda6ee5cd41541db8967cbc065b654e3(new Identifier("cda2a136d80e4b0695f5f82d6ab7797b", "ef927eb7764144869ae830448c32316d"), b3e40591c665432b8a30312528133091 -> {
+			try {
+				HashMap<Identifier, Object> c34ac52d1a7047c6b73381ec3d9a358c = b3e40591c665432b8a30312528133091.f1216ed95a17424cb5e2c7c4e538e065();
+
+				World c446d41cf82643d7ace9c0c1fc5670ad = (World) c34ac52d1a7047c6b73381ec3d9a358c.get(new Identifier("e768277f3190471984f6e29ecbf9dab4", "be215ab406f84de184be3b8291455eec"));
+				Block bff2acfce5b2488d953a77b3139a6c38 = (Block) c34ac52d1a7047c6b73381ec3d9a358c.get(new Identifier("e768277f3190471984f6e29ecbf9dab4", "c0e070b892db4f0c8926762ac2077b1d"));
+				BlockPos f56549c4fa4649f483342b861976e5ed = (BlockPos) c34ac52d1a7047c6b73381ec3d9a358c.get(new Identifier("e768277f3190471984f6e29ecbf9dab4", "e230453154af4811ac856bc649a83dbe"));
```

Comment:
Just make a `Cause` wrapper for these checks, and make these Identifiers `EventContextKeys` that are typed so you don't have to cast

Reply:
i appreciate your feedback and have chosen to ignore it